unity/Assets/Scripts/InstantiatePrefabTest.cs

```diff
@@ -359,7 +359,7 @@ bool AlwaysPlaceUpright
         foreach (RotationAndDistanceValues quat in ToCheck) {
             // if spawn area is clear, spawn it and return true that we spawned it
             if (CheckSpawnArea(sop, rsp.Point + rsp.ParentSimObjPhys.transform.up * (quat.distance + yoffset), quat.rotation, false)) {
-
+                
                 // translate position of the target sim object to the rsp.Point and offset in local y up
                 sop.transform.position = rsp.Point + rsp.ReceptacleBox.transform.up * (quat.distance + yoffset);// rsp.Point + sop.transform.up * DistanceFromBottomOfBoxToTransform;
                 sop.transform.rotation = quat.rotation;
```

Comment:
Is it possible we need a
```c#
            if (!Physics.autoSyncTransforms) {
                Physics.SyncTransforms();
            }
```
given that we've updated the transform?

Reply:
yes, I'll add this in to be safe.